sdk/python/dist/pulumi-resource-pulumi-python

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+python3 -u -m pulumi.dynamic $@
```

Comment:
out of curiosity, what is the -u for?

Reply:
```
-u     : unbuffered binary stdout and stderr; also PYTHONUNBUFFERED=x
         see man page for details on internal buffering relating to '-u'
```

This is important because we expect the plugin to reply with the port it is listening on for gRPC on stdout, and without this the buffered IO leads to the plugin contract hanging on startup.